cmux-tui/scripts/test_verify_published_manifest.py

```
@@ -0,0 +1,264 @@
+                        "--require-artifact",
+                        self.WINDOWS,
+                    ]
+                )
+        self.assertEqual(result, 1)
+        self.assertEqual(stdout.getvalue(), "")
+        self.assertEqual(stderr.getvalue(), "cmux-tui manifest verification failed\n")
+        self.assertNotIn(url, stdout.getvalue() + stderr.getvalue())
+        self.assertNotIn(secret, stdout.getvalue() + stderr.getvalue())
+
+    def test_rejects_non_https_manifest_url_before_fetching(self) -> None:
+        with self.assertRaisesRegex(VERIFY.ManifestError, "HTTPS"):
```

Ruff (0.16.1)

[warning] Use `pytest.raises` instead of unittest-style `assertRaisesRegex`

Replace `assertRaisesRegex` with `pytest.raises`

(PT027)